renderer-macos/lib/stubs/BriskButton.c

```diff
@@ -36,18 +36,19 @@
 }
 
 - (void)setCallback:(value)callback {
```

Comment:
Almost all callbacks should handle `option(callback)` case IMO.

Reply:
Filed under #76.